packages/webdoc-default-template/tmpl/components/members.tmpl

```diff
@@ -0,0 +1,48 @@
+<?js
+const
+  categoryData = obj,
```

Comment:
Again, description on top.

Reply:
Fixed in 9e7db62055baad1b7af9e1bcbdbfbd3fa02c5520.